buildroot/share/VS_code/AutoBuildVScode/README.md

```diff
@@ -0,0 +1,66 @@
+# Auto Build support for VS code
+
+This `VS code` extension provides access to the `Auto Build` script.
+
+## Installation
+
+### 1. Install VS code
+
+Pointers to setup instructions for the supported platforms are near the top of [Setting up Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview).
+
+
+### 2. Install this extension
+
+Copy the directory `buildroot/share/VS_code/AutoBuildVScode` to the `VS code` extension directory and then (re)start `VS code`.
+
+The `VS code` extension directory is usually located at:
+- Windows - C:/Users/YOUR_USER_NAME/.vscode/extensions
+- Mac - tbd
+- Linux - tbd
```

Comment:
Apparently it can also be [packaged as a VSIX file](https://code.visualstudio.com/docs/extensions/publish-extension#_packaging-extensions) for installation through the VSCode interface.

Reply:
That appears to be for making an extension available to everyone.  The Auto Build script is only useful with Marlin.  I don't see any value in making it generally available.

Reply:
A VSIX file can be made to give to your friends an extension in a convenient installable form without publishing it for all.